models/staging/tpdm/stage/stg_tpdm__evaluation_ratings.sql

```diff
@@ -0,0 +1,26 @@
+with base_evaluation_ratings as (
+    select * from {{ ref('base_tpdm__evaluation_ratings') }}
+    where not is_deleted
+),
+keyed as (
+    select
+        {{ dbt_utils.surrogate_key(
+            [
+                'tenant_code',
+                'api_year'
```

Comment:
Here we have two references in API response and I do not know which one I should take to extarct value. We discussed it on daily and decided leave is as it is, but we have extract values to create dedup_key for stage models
![image](https://github.com/user-attachments/assets/42b7545b-2332-480f-9ef8-edb6adaf2ec3)